### ISSUE TITLE
Ability to use without a target

### ISSUE THREAD

**Issue by hstonge** [auchenberg/chrome-devtools-app#45]
It would be great if this app could be launched without requiring a target, so one could use certain features like the heap snapshot file analysis/comparison, which doesn't really require a target in the first place.

**Comment by auchenberg** [comment]
That's a great idea! Simply start the app with DevTools loaded?

**Comment by hstonge** [comment]
I could be that way. Or perhaps have a small text below the "splash" image with a link/button to open the DevTools directly (possibly with a checkbox suggesting "Always open the DevTools at launch").

**Comment by auchenberg** [comment]
Yeah, that could work. I'm currently swarmed with work, so if you have 10mins to add the button, it could be much appreciated, otherwise I'll have a look in the new year.

**Comment by hstonge** [comment]
Oh don't worry, it's only a suggestion, nothing urgent ;)
